Add support for CentOS Stream 10

psathyan opened the pull request:
Include the support for `CentOS Stream 10`, this is also avilable as part of infra.

kalebskeithley commented:
agree with adding stream10 (10s).   We don't care about or need aarch64 or powerppc.

psathyan committed:
Add support for reserving CentOS Stream 10

Including a choice to add support for CentOS Stream 10. This
option is available to us as part of the infra.

Signed-off-by: Pragadeeswaran Sathyanarayanan <pragadeeswaran.sathyanarayanan@ibm.com>

sachinpunadikar reviewed (approved):
LGTM
